### DISCUSSION TITLE
Welcome to machine-learning-interview Discussions!

### DISCUSSION THREAD

**Discussion by khangich** [khangich/machine-learning-interview#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about MLE interviews. 
  * Share ideas on what content should be cover. 

  To get started, comment below about what questions you have about MLE interviews

**Comment by yashsandansing** [comment]
Why is "Java Garbage Collection" included in the programming section instead of "Python Garbage Collection"?
